### ISSUE TITLE
Battery and network status in menus?

### ISSUE THREAD

**Issue by fennectech** [RSDuck/vba-next-switch#34]
Since we cannot return to the home menu while playing homebrew we should add battery and wireless status to the margins of the emulator and menus.   As well as make the margins match the color of horizon.

**Comment by xne0n** [comment]
yes why not. But did you know you can long press home button to display the quick setting menu on the right which include all of that. And you can press b or touch the screen to quit it .

**Comment by fennectech** [comment]
Its really neat that we have such nice sized margins   Imagine the stuff we could cram into there!
